Build/Src/FwBuildTasks/FwBuildTasks.csproj

```diff
@@ -3,13 +3,13 @@
     <RootNamespace>SIL.FieldWorks.Build.Tasks</RootNamespace>
     <Description>Additional msbuild tasks for FieldWorks</Description>
     <Product>FwBuildTasks</Product>
-    <TargetFramework>net461</TargetFramework>
+    <TargetFramework>net462</TargetFramework>
     <OutputPath>../..</OutputPath>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
   </PropertyGroup>
-
   <ItemGroup>
     <PackageReference Include="NUnit" Version="3.13.3" PrivateAssets="All" />
+    <PackageReference Include="NUnit3TestAdapter" Version="4.6.0" />
```

Comment:
I would double check this version of the adapter works, I believe only `4.3.2` supports this version of NUnit but I may be thinking of the dotnet version in which case I'm not sure if version `4.3.2` is only required for net461

Reply:
Yes it did work, adding this allowed John to run the tests in visual studio without Resharper. I bumped this up to 462 since it was missed in the earlier round of updates. It looks like NUnit3TestAdapter at this version supports 462.
<!-- Reviewable comment -OE0ahbs0xmFJF6FGDfP:b-vz48pp -->
<!-- Sent from Reviewable.io -->